### PR TITLE
Update links from docs to learn

### PR DESCRIPTION
**Pull request by lukemurraynz**
Update links from docs to learn. Based on recent Microsoft Learn changes, to update alias.

### CODE REVIEW

**Review by xstof — APPROVED** [review]
Looks good to me.

**Comment by erjosito** [comment]
Wow @lukemurraynz Thanks a lot! I hope you have done this programmatically :-D There are some conflicts, probably because we merged other PRs first. Could you please resolve the conflicts, maybe running your magic on the latest repo version?

**Comment by lukemurraynz** [comment]
> Wow @lukemurraynz Thanks a lot! I hope you have done this programmatically :-D There are some conflicts, probably because we merged other PRs first. Could you please resolve the conflicts, maybe running your magic on the latest repo version?

Yep! - Those two files seem to be causing an issue; I'll deal with them separately as part of another PR after this has been committed.

File: SECURITY.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Microsoft takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [Microsoft](https://github.com/Microsoft), [Azure](https://github.com/Azure), [DotNet](https://github.com/dotnet), [AspNet](https://github.com/aspnet), [Xamarin](https://github.com/xamarin), and [our GitHub organizations](https://opensource.microsoft.com/).
 
-If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
+If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://learn.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
```
Comment (johndowns) [inline SECURITY.md:7]: Just a minor thing - we shouldn't link to locale-specific articles. (I know this was a pre-existing problem!) Would you please be able to fix this as part of the same PR?

```suggestion
If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://learn.microsoft.com/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
```

Comment (lukemurraynz) [reply SECURITY.md]: Done. I have sorted the en-us links - will just check other languages.

Comment (lukemurraynz) [reply SECURITY.md]: Done.

Comment (johndowns) [reply SECURITY.md]: Thank you!

**Comment by erjosito** [comment]
There we go!

**Pull request merged**